src/scenes/home/team/team.js

```diff
@@ -1,31 +1,36 @@
 import React, { Component } from 'react';
-import axios from 'axios';
 import Section from 'shared/components/section/section';
 import QuoteBanner from 'shared/components/quoteBanner/quoteBanner';
-import TeamCard from 'shared/components/teamCard/teamCard';
 import BoardCard from 'shared/components/boardCard/boardCard';
+import TeamCard from 'shared/components/teamCard/teamCard';
 import BoardMembers from './boardMembers';
 import styles from './team.css';
 
+
 class Team extends Component {
   state = {
-    members: [],
+    members: [
```

Comment:
When we have a bunch of JSON data like this, I'd prefer for it to be external and imported in. Just like I have it with exec.